wellcomecollection/wellcomecollection.org
# Updown: Upgrade packages to avoid security issues

## What does this change?

Fixes flagged security issues:
High:
https://github.com/wellcomecollection/wellcomecollection.org/security/dependabot/187

Moderate:
https://github.com/wellcomecollection/wellcomecollection.org/security/dependabot/110
https://github.com/wellcomecollection/wellcomecollection.org/security/dependabot/109

## How to test

`cd updown`
`rm -rf node_modules && rm -rf yarn.lock`
`yarn`
Follow README doc to run `apply`, should work nicely and use updated versions of the problematic dependencies listed above.

## How can we measure success?

Enhanced security

## Have we considered potential risks?
N/A
